Sources/CardPayments/APIRequests/ConfirmPaymentSourceRequest.swift

```diff
@@ -31,7 +31,8 @@ struct ConfirmPaymentSourceRequest: APIRequest {
         confirmPaymentSource.paymentSource = PaymentSource(card: card)
         
         self.orderID = cardRequest.orderID
-        self.accessToken = accessToken
+        let modifiedClientID = clientID + ":"
+        self.base64EncodedClientID = Data(modifiedClientID.utf8).base64EncodedString()
```

Comment:
```suggestion
        self.base64EncodedClientID = Data(clientID.appending(":").utf8).base64EncodedString()
```

This more similarly matches the Android implementation as well

Reply:
Fixed in this commit: 2c4f78ac2eaffce9ec89d88e655f2ccb1b04e096